packages/aws-cdk-lib/aws-ec2/README.md

```diff
@@ -973,6 +973,7 @@ new ec2.VpcEndpointService(this, 'EndpointService', {
   vpcEndpointServiceLoadBalancers: [networkLoadBalancer1, networkLoadBalancer2],
   acceptanceRequired: true,
   allowedPrincipals: [new iam.ArnPrincipal('arn:aws:iam::123456789012:root')]
+  contributorInsights: true
```

Comment:
Can you please add a short description of the new property in the above paragraph?

Reply:
done.